index.html

```diff
@@ -2467,154 +2467,153 @@ <h3>The <dfn>NFCReaderOptions</dfn> dictionary</h3>
+                of |ndef|'s <a>PAYLOAD field</a> to the result of running <a>ASCII encode</a>
+                on |language|.
               </li>
               <li>
-                Set position from <var>offset</var> + 1 to end, of the <a>byte sequence</a>
-                of <var>ndef</var>'s <a>PAYLOAD field</a> to the result of running <a>UTF-8 encode</a>
-                on <var>record</var>'s data.
+                Set position from |offset| + `1` to end, of the <a>byte sequence</a>
+                of |ndef|'s <a>PAYLOAD field</a> to the result of running <a>UTF-8 encode</a>
+                on |record|'s data.
               </li>
             </ol>
           </li>
           <li>
-            Return <var>ndef</var>.
+            Return |ndef|.
           </li>
         </ol>
       </p>
       </section>
 
       <section><h3>Mapping URL to NDEF</h3>
       <p>
-        To <dfn>map a URL to NDEF</dfn> given a <var>record</var>, run these
+        To <dfn>map a URL to NDEF</dfn> given a |record:NDEFRecordInit|, run these
         steps:
         <ol>
           <li>
-            If <var>record</var>'s data is not a <a>string</a>,
-            [= exception/throw =] a {{"TypeError"}} and abort these steps.
+            If |record|'s data is not a {{DOMString}},
+            [= exception/throw =] a {{TypeError}} and abort these steps.
           </li>
           <li>
-            Let <var>urlRecord</var> be the result of
-            <a data-lt="url parser">parsing</a> <var>record</var>'s data.
+            Let |urlRecord:USVString| be the result of
+            <a data-lt="url parser">parsing</a> |record|'s data.
           </li>
           <li>
-            If <var>urlRecord</var> is failure, [= exception/throw =] a
+            If |urlRecord| is failure, [= exception/throw =] a
             {{"SyntaxError"}} {{DOMException}} and abort these steps.
           </li>
           <li>
-            Let <var>ndef</var> be the notation for the <a>NDEF record</a> to
+            Let |ndef| be the notation for the <a>NDEF record</a> to
             be created by the UA.
           </li>
           <li>
-            Set the <var>ndef</var>'s <a>TNF field</a> to 1 (<a>well-known type record</a>).
+            Set the |ndef|'s <a>TNF field</a> to `1` (<a>well-known type record</a>).
           </li>
           <li>
-            Set the <var>ndef</var>'s <a>TYPE field</a> to "U" (0x55
+            Set the |ndef|'s <a>TYPE field</a> to "U" (0x55
             following NFC binary encoding).
           </li>
           <li>
             Match the URI prefixes as defined in [[[NFC-STANDARDS]]]
             URI Record Type Definition specification, Section 3.2.2, against
-            the <var>urlRecord</var>.
+            the |urlRecord|.
           </li>
           <li>
-            Let <var>prefix</var> be the matched prefix or else the
+            Let |prefix:string| be the matched prefix or else the
```

Comment:
Is string correct? Or should we use USVString @marcoscaceres

Reply:
I'm not sure what prefix is in this context... if it's part of a URL, then a USVString might be suitable... if it's just a random string, then string is fine.